references/common/application.go

```diff
@@ -90,6 +90,40 @@ type DeleteOptions struct {
 	ForceDelete bool
 }
 
+// ScaleOptions is options for scale
+type ScaleOptions struct {
+	Namespace string
+	AppName   string
+	CompName  string
+	Replicas  int64
+	C         common.Args
+	Client    client.Client
+}
+
+// ScaleComponent will scale component including server side
+func (o *ScaleOptions) ScaleComponent(compName string, replicas int64, io cmdutil.IOStreams) error {
+	ctx := context.Background()
+	app := new(corev1beta1.Application)
+	err := o.Client.Get(ctx, client.ObjectKey{Name: o.AppName, Namespace: o.Namespace}, app)
+	if err != nil {
+		return client.IgnoreNotFound(err)
+	}
+	for ci, comp := range app.Spec.Components {
+		if comp.Name == compName {
+			for ti, trait := range comp.Traits {
+				if trait.Type == "scaler" {
+					raw := fmt.Sprintf("{\"replicas\":%d}", replicas)
+					app.Spec.Components[ci].Traits[ti].Properties.Raw = []byte(raw)
+				}
+			}
```

Comment:
It is better that print some notice message to replace the success log if the component does not have the scaler trait.

Reply:
I have fixed this problem. If can't find scaler trait, it will throw err "Cannot find scaler trait in this component.Please specify scaler trait."